dockers/docker-orchagent/supervisord.conf.j2

```diff
@@ -152,7 +178,9 @@ stdout_logfile=syslog
 stderr_logfile=syslog
 dependent_startup=true
 dependent_startup_wait_for=swssconfig:exited
+{% endif %}
 
+{% if is_fabric_asic == 0 %}
```

Comment:
The concern with this approach is, if someone have to add a new process in the end, he may ignore this `if is_fabric_asic`. My suggestion is lets have one section for `is_fabric_asic == 0` and else section for fabric processes (similar to critical_process.j2). This way priority can also be adjusted accordingly. What do you think?

Reply:
Sure, that makes sense.

Reply:
@prsunny I addressed your comments. Thanks for reviewing. The new commit 1 overrided the previous commit when I resolved conflicts. Hope that it's clear to you. Also updated the title + description.

Reply:
I would prefer to have it properly listed in if-else. Currently the order/priority is not in order. This would be more confusing if we have to add new process later. please keep the original order for `if not-fabric` and add the fabric process in `else `part

Reply:
Sounds reasonable.  But some processes like `orchagent` is available in both `fabric` and `not fabric` part. So if we are doing that:

`if not-fabric:`
`    orchagent`
`    portsyncd`
`else:`
`    orchagent`
`end-if`

`orchagent` will appear both, and seems redundant. Is this OK? All processes for fabric are also in `not fabric`

Reply:
My main concern that having `if` and `else` requires to update `orchagent` at both parts if someone wants to update `orchagent`'s configuration. That seems harder to manage and redundant. So having `if` may be better.